xwiki-platform-core/xwiki-platform-eventstream/xwiki-platform-eventstream-stores/xwiki-platform-eventstream-store-solr/src/main/java/org/xwiki/eventstream/store/solr/internal/SolrEventStore.java

```diff
@@ -677,7 +677,16 @@ private String serializeCompareCondition(CompareQueryCondition condition)
 
         switch (condition.getType()) {
             case EQUALS:
-                builder.append(toFilterQueryString(condition.getProperty(), condition.getValue()));
+                String queryString = toFilterQueryString(condition.getProperty(), condition.getValue());
+                if (queryString == null) {
+                    // See https://stackoverflow.com/a/28859224.
```

Comment:
A short summary of the content of the link could be interesting imo.

Reply:
Yes, I'm not a fan of giving stackoverflow links in comment, I much prefer an explanation and more official references.

Reply:
Well, imo having any kind of link is fine, but it not self-sufficient as links can deprecate quite quickly.